enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -576,10 +584,17 @@ func NewPebbleCache(env environment.Env, opts *Options) (*PebbleCache, error) {
 		includeMetadataSize:         opts.IncludeMetadataSize,
 	}
 
-	versionMetadata, err := pc.databaseVersionMetadata()
+	versionMetadata, err := pc.DatabaseVersionMetadata()
 	if err != nil {
 		return nil, err
 	}
+	if created && *opts.ActiveKeyVersion < 0 {
+		versionMetadata = pc.maxDatabaseVersionMetadata()
+	} else if created {
+		versionMetadata.MinVersion = *opts.ActiveKeyVersion
+		versionMetadata.MaxVersion = *opts.ActiveKeyVersion
```

Comment:
Do we have to set LastModifyUsec in this case?

Reply:
I don't think it's necessary, but it's probably slightly more correct so I went ahead and added it. Good call!